src/settings.ts

```diff
@@ -16,10 +17,12 @@ const defaultSettings: Settings = {
 type TempUnit = "celsius" | "fahrenheit";
 type ClockType = "12" | "24";
 type FontType = "thin" | "bold";
+type ColorScheme = "default" | "light" | "dark" | "custom";
```

Comment:
I think that `system` instead of `default` will better indicate what it represents.